docs/mvc/tutorials/mvc-with-entity-framework/sample/src/ContosoBooks/Controllers/AuthorController.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Mvc;
+using ContosoBooks.Models;
+using Microsoft.Framework.Logging;
+using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.Data.Entity.Storage;
+using Microsoft.Data.Entity;
+
+// For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace ContosoBooks.Controllers
+{
+    public class AuthorController : Controller
+    {
+        [FromServices]
+        public BookContext DbContext { get; set; }
```

Comment:
Minor thing... I would probably call this property BookContext as it is a particular type of a DbContext (it's not unusual to have more than one type of context in your app.